style.css

```diff
@@ -352,3 +402,212 @@ form ::placeholder {
   font-size: 15px;
   line-height: 24px;
 }
+
+.menu ul {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .homepage {
+    background-image: url(images/Header\ bg.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
```

Comment:
Kindly add border radius to the left bottom of your homepage area. Check the differences below.
<img width="1121" alt="Screenshot 2023-01-19 at 10 38 12 PM" src="https://user-images.githubusercontent.com/25887253/213543693-843a7b32-8ab7-4187-b4d4-a8b3cbd7bd5b.png">
<img width="565" alt="Screenshot 2023-01-19 at 10 38 37 PM" src="https://user-images.githubusercontent.com/25887253/213543700-436afad2-9901-4e58-b18d-875f2bd5d1de.png">